hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/context/properties/BootstrapPropertiesTest.java

```diff
@@ -522,17 +522,17 @@ class BootstrapPropertiesTest {
             entry(TOKENS_NFTS_MAX_BATCH_SIZE_MINT, 10),
             entry(TOKENS_NFTS_MAX_BATCH_SIZE_BURN, 10),
             entry(TOKENS_NFTS_MAX_METADATA_BYTES, 100),
-            entry(TOKENS_NFTS_MAX_ALLOWED_MINTS, 5000000L),
+            entry(TOKENS_NFTS_MAX_ALLOWED_MINTS, 10_000_000L),
             entry(TOKENS_NFTS_MINT_THORTTLE_SCALE_FACTOR, ScaleFactor.from("5:2")),
-            entry(TOKENS_NFTS_USE_VIRTUAL_MERKLE, false),
+            entry(TOKENS_NFTS_USE_VIRTUAL_MERKLE, true),
```

Comment:
Is there an equivalent for fungible tokens?

Reply:
@poulok no, we don't have a flag for fungible token